frontend/src/metabase/query_builder/containers/QueryBuilder.unit.spec.tsx

```diff
@@ -59,7 +59,7 @@ const TEST_MODEL_CARD = createMockCard({
   },
   dataset: true,
   display: "scalar",
-  description: "Test description", 
+  description: "Test description",
```

Comment:
hmm, how was that possible? wasn't it formatted by prettier?

Reply:
Yeah, I just wanted to retrigger the tests since this file is failing on Github Actions but successful on my local machine 🥲

Reply:
ah, it's fine. just in case, you can create an empty commit using `git commit -m 'trigger ci' --allow-empty`

also there should be a button to restart tests via UI

Reply:
but I'd suggest you to rebase first 😄

Reply:
Hehehe let's try this again then!